linode/acceptance/util.go

```diff
@@ -570,8 +570,12 @@ func ModifyProviderMeta(provider *schema.Provider, modifier ProviderMetaModifier
 	}
 }
 
-// GetRegionsWithCaps returns a list of regions that support the given capabilities.
-func GetRegionsWithCaps(capabilities []string, filters ...RegionFilterFunc) ([]string, error) {
+// GetRegionsWithCaps returns a list of region IDs that support the given capabilities
+// Parameters:
+// - capabilities: Required capabilities that the regions must support.
+// - siteType: The site type to filter by ("core" or "distributed").
+// - filters: Optional custom filters for additional criteria.
+func GetRegionsWithCaps(capabilities []string, regionType string, filters ...RegionFilterFunc) ([]string, error) {
```

Comment:
Would it make sense to also accept `any` as a regionType for situations where we aren't picky about core vs distributed?

Reply:
Good call, I will add option to give `any` as an argument for `regionType`